buildout.cfg

```diff
@@ -91,6 +91,8 @@ aws_ip_ranges_path = ${buildout:directory}/aws-ip-ranges.json
 elasticsearch.server = ${buildout:es-ip}:${buildout:es-port}
 file_upload_profile_name = encoded-files-upload
 ontology_path = ${buildout:directory}/ontology.json
+external_aws_s3_transfer_allow = false
+external_aws_s3_transfer_buckets = ${buildout:directory}/.aws/direct-external-s3-list
```

Comment:
The 'direct-external-s3-list' is in the s3 bucket folder that gets downloaded for every build.  It seemed like a good place to store the list of external s3 buckets that need the external transfer functionality.